front/src/applications/editor/tools/pointEdition/components/PointEditionLeftPanel.tsx

```diff
@@ -72,24 +72,22 @@ const PointEditionLeftPanel = <Entity extends EditorEntity>({
 
   useEffect(() => {
     const firstLoading = trackState.type === 'idle';
-    const trackId = state.entity.properties.track as string | undefined;
+    const trackId = state.entity.properties.track;
```

Comment:
Yes that's not a big deal. With some few `any`'s in the app, I feel scared to pass an any value to an object :) For me the cast was there to attest that we checked/tested the data from this const and we know it can be one of these 2.

But I understand why you would prefer to "break" the app if we misjudged the type (especially if it changes later and we forget to update this part).

Reply:
Hm, I hadn't noticed that `state.entity.properties.track` had the type `any`… Then yeah I agree being explicit here is a good idea. Changed.

Note that there are many other places where we use `entity.properties.XXX` and don't cast or anything (so `any` spreads via type inference). We should probably enable the `no-unsafe-assignment` lint to avoid these footguns (see #8772).

Reply:
Yeah lots of `any`s in `geojson` and stuff... :)